test/hotspot/jtreg/runtime/ClassFile/UnsupportedClassFileVersion.java

```diff
@@ -27,8 +27,7 @@
  * @modules java.base/jdk.internal.org.objectweb.asm
  *          java.base/jdk.internal.misc
  *          java.management
- * @compile -XDignore.symbol.file UnsupportedClassFileVersion.java
```

Comment:
I can't find what -XDignore.symbol.file was used for, so am unsure if removing it is in fact okay in all circumstances. ??

Reply:
I believe we had to have it b/c we used JDK internal API, i.e. asm, now we have corresponding `@modules` tags, and `-XDignore.symbol.file` isn't need anymore. also, it's used at `@compile`, so if it is required, the test would fail fast and (largely) independently from JVM configurations.

Reply:
Okay, it seems it is not needed on this test as there are no warnings without it. Thanks.